packages/mask/src/plugins/Trader/trader/uniswap/usePairs.ts

```diff
@@ -31,7 +31,7 @@ export function usePairs(tradeProvider: TradeProvider, tokenPairs: readonly Toke
         if (!FACTORY_CONTRACT_ADDRESS || !INIT_CODE_HASH) return EMPTY_LIST
         return tokenPairs.map(([tokenA, tokenB]) =>
             tokenA && tokenB && !tokenA.equals(tokenB)
-                ? getPairAddress(FACTORY_CONTRACT_ADDRESS, INIT_CODE_HASH, tokenA, tokenB)
+                ? getPairAddress(FACTORY_CONTRACT_ADDRESS, INIT_CODE_HASH, tokenA, tokenB, stable)
```

Comment:
A property passing down is unnecessary. Prefer to use `context.STABLE`. Yes, treat the `STABLE` as a common configuration of trade providers.

Reply:
Changed